api-matlab/rtQC_display_setup.m

```diff
@@ -964,26 +964,48 @@
     'Style', 'push',...
     'String', 'Show FD outliers', ...
     'Units', 'Normalized',...
-    'Position', [0.8 0.7 0.18 0.3],...
+    'Position', [0.8 0.795 0.18 0.225],...
     'CallBack', @showFDoutliers, ...
     'UserData', 0,...
     'fontsize', gui_data.button_font_size);
 gui_data.pb_run_qc_post = uicontrol('Parent', gui_data.panel_rtsummary,...
     'Style', 'push',...
     'String', 'Run Offline QC', ...
     'Units', 'Normalized',...
-    'Position', [0.8 0.38 0.18 0.3],...
+    'Position', [0.8 0.55 0.18 0.225],...
     'CallBack', @runOfflineQC, ...
     'UserData', 0,...
     'fontsize', gui_data.button_font_size);
 gui_data.pb_open_qc_post = uicontrol('Parent', gui_data.panel_rtsummary,...
     'Style', 'push',...
     'String', 'Generate report', ...
     'Units', 'Normalized',...
-    'Position', [0.8 0.06 0.18 0.3],...
+    'Position', [0.8 0.06 0.18 0.225],...
     'CallBack', @generateReport, ...
     'UserData', 0,...
     'fontsize', gui_data.button_font_size);
+gui_data.pb_get_physio_batch = uicontrol('Parent', gui_data.panel_rtsummary,...
```

Comment:
The positioning of the button and dropdown on the GUI is not great, when run on my Mac:
<img width="299" alt="Screen Shot 2020-01-07 at 11 04 08" src="https://user-images.githubusercontent.com/10141237/71888387-53b10700-3140-11ea-85e1-f74a2166f43c.png">

@NinaWie  Does is render differently on your system?

Previously I haven't given much attention to cross-platform functionality, at least in the sense that the display of the GUI and buttons are fine. Maybe that needs some more attention.

For this specific case, maybe check if you can update the position of these UIcontrols, and perhaps others on the same tab and group, such that the layout is user friendly.

Reply:
Yes, it renders perfectly fine on my system. But I now changed it such that "select vendor" is inside the drop down menu, which I like better than the label above the menu anyway. You can try again, the new commit should be part of the pull request already. Let me know if it's still ugly on the mac.

Reply:
Perfect